src/utils.js

```diff
@@ -112,3 +60,84 @@ export function buildClause (field, value, opts) {
 
   return Object.assign({}, mainClause, opts)
 }
+
+export function toBool (filters) {
+  const unwrapped = {
+    must: unwrap(filters.and),
+    should: unwrap(filters.or),
+    must_not: unwrap(filters.not),
+    minimum_should_match: filters.minimum_should_match
+  }
+
+  if (
+    filters.and.length === 1 &&
+    !unwrapped.should &&
+    !unwrapped.must_not
+  ) {
+    return unwrapped.must
+  }
+
+  const cleaned = {}
+
+  if (unwrapped.must) {
+    cleaned.must = unwrapped.must
+  }
+  if (unwrapped.should) {
+    cleaned.should = filters.or
+  }
+  if (unwrapped.must_not) {
+    cleaned.must_not = filters.not
+  }
+  if (
+    unwrapped.minimum_should_match &&
+    unwrapped.minimum_should_match <= filters.or.length
```

Comment:
So this prevents setting a `minimum_should_match` that's larger than the number of `should`s, right? I see this as an optional sanity check, but sounds good to include it.